osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -263,7 +263,8 @@ private void load(OsuColour colours, LocalisationManager localisation, BeatmapDi
                                 Shear = -wedged_container_shear,
                                 TextSize = 11,
                                 TextPadding = new MarginPadding { Horizontal = 8, Vertical = 2 },
-                                Status = beatmapInfo.Status,
+                                // TODO: remove case once IBeatmapInfo somehow provides this information (needs RealmBeatmap implementation).
+                                Status = ((BeatmapInfo)beatmapInfo).Status,
```

Comment:
While I get that there's a todo item there, I'm not sure it makes sense to do this even temporarily, given the context of this pull. Like if the goal is to migrate song select to realm models via scoping everything up from EF models to the interface, this will have to be resolved before that is fully done anyway, lest song select crash/fail to display the wedges at runtime anyways...

I guess if the purpose is to unblock the progress of purely mechanical changes without addressing the storage concerns of online stuff yet, I could be fine with this, temporarily.

Reply:
Can agree with that. I have quite a few of these in ongoing work and have generally avoided PRing them because of the reasons you state.

If preferred I can close this PR for now, convert it to a safe cast, or block this PR until storage is added to realm (might look at that today).

Reply:
If it works out with your plans I would definitely prefer the last option (block until storage is added). I have a bit of a worry that leaving a safe cast is going to lead to it being forgotten about and cause a relatively subtle regression. Because of that a hard crashing cast would be counterintuitively preferable to me, as at least that has a better chance to be noticed/fail CI.